From d420b3f06c788364052c9a6700812516db0d637d Mon Sep 17 00:00:00 2001
From: Ken Rockot <rockot@google.com>
Date: Mon, 8 Jan 2024 13:14:29 -0800
Subject: [PATCH] Suppress gn check for absl CHECK dependency

Rather than allow protected absl target visibility to mediapipe code
for a proper log dependency, suppress GN check for the one instance
where it would be needed. This is just a header dependency.
---
 .../mediapipe/src/mediapipe/framework/calculator_graph.cc       | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/third_party/mediapipe/src/mediapipe/framework/calculator_graph.cc b/third_party/mediapipe/src/mediapipe/framework/calculator_graph.cc
index 1bd356eac46f9..b606652f3d744 100644
--- a/third_party/mediapipe/src/mediapipe/framework/calculator_graph.cc
+++ b/third_party/mediapipe/src/mediapipe/framework/calculator_graph.cc
@@ -28,7 +28,7 @@
 #include "absl/container/flat_hash_set.h"
 #include "absl/log/absl_check.h"
 #include "absl/log/absl_log.h"
-#include "absl/log/check.h"
+#include "absl/log/check.h"  // nogncheck
 #include "absl/memory/memory.h"
 #include "absl/status/status.h"
 #include "absl/strings/str_cat.h"
-- 
2.43.0.195.gebba966016-goog

